shell/README.md

```diff
@@ -0,0 +1,51 @@
+Shell
+=====
```

Comment:
Old style?

Reply:
Yes! My plan is to update the style across the board in #617. I think the style changes will be easier to review in a standalone PR that doesn't move any files or change any content.